src/pages/home.tsx

```diff
@@ -1,23 +1,79 @@
 import { LinkButton } from "~components/Button";
 import { useRecentEvents } from "~utils/hooks/history";
+import { useEffect, useState } from "react";
+import Markdown from "react-markdown";
+import pjson from "../../package.json";
+import "./markdown.css";
 
 export const HomePage: React.FC = () => {
   const { data: recentEvents } = useRecentEvents(5);
 
+  const [markdownContent, setMarkdownContent] = useState<string>("");
+
+  useEffect(() => {
+    // Fetch the update notes as a markdown file
+    const fetchMarkdownContent = async () => {
+      try {
+        const response = await fetch("updateNotes.md");
```

Comment:
The fetch here should be an absolute path (`await fetch("/updateNotes.md")`), and you should probably put the file in `public`